api/src/routers/meals.js

```diff
@@ -0,0 +1,70 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const router = express.Router();
+
+// router to get all the meals
+router.get("/", async (req, res, next) => {
+  try {
+    const meals = await knex.select("*").from("Meal");
+    res.json(meals);
+  } catch (error) {
+    next(error);
+  }
+});
+
+// router to create a new meal
+router.post("/", async (req, res, next) => {
+  try {
+    const newMeal = req.body;
+    const [mealId] = await knex("Meal").insert(newMeal);
```

Comment:
You named your table `Meal`, checkout this page for SQL table/column name conventions: [https://brainstation.io/learn/sql/naming-conventions](https://brainstation.io/learn/sql/naming-conventions)

```suggestion
    const [mealId] = await knex("meal").insert(newMeal);
```